### ISSUE TITLE
maptalks is not defined  in requirejs

### ISSUE THREAD

**Issue by lifefriend** [maptalks/maptalks.js#628]
**maptalks's version and what browser you use?**

 maptalks-0.38.2

**Issue description**

when edit rectangle in requirejs 
```
Uncaught ReferenceError: maptalks is not defined
   at Circle.getOutline (maptalks.js:11969)
    at GeometryEditor._createOrRefreshOutline (maptalks.js:21887)
    at GeometryEditor.start (maptalks.js:21795)
    at Circle.startEdit (maptalks.js:23170)
```
change "return new maptalks.Polygon" to "return new Polygon" can fix it in function "Path.prototype.getOutline"

**Please provide a reproduction URL (on any jsfiddle like site)**

**Comment by fuzhenn** [comment]
Confirmed, really thanks!

**Comment by fuzhenn** [comment]
Fixed and published as v0.39.1
Thanks again!
